config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits.yaml

```diff
@@ -170,3 +170,27 @@ presubmits:
       testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
       testgrid-tab-name: verify-boilerplate
       testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
+  - name: pull-cluster-api-provider-azure-conformance-v1alpha3
+    always_run: false
+    optional: true
+    decorate: true
+    labels:
```

Comment:
Doesn't this also need:
```
    extra_refs:
    - org: kubernetes
      repo: kubernetes
      base_ref: master
      path_alias: k8s.io/kubernetes
```
for `build_k8s()`?

Reply:
indeed, I didn't see it in the aws equivalent job config though ?

Reply:
The CAPA jobs are a bit confusing since we have two ways of running conformance, this is the one that is running the equivalent script: https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml#L64

Reply:
thanks, added